apisix/upstream.lua

```diff
@@ -558,6 +559,26 @@ local function filter_upstream(value, parent)
         end
         value.nodes = new_nodes
     end
+
+    if value.checks then
+        core.log.info("guohao test will ensure check")
+        if healthcheck == nil then
+            healthcheck = require("resty.healthcheck")
+        end
+        local name = get_healthchecker_name(value.parent)
+        local upstream_target_list = healthcheck.get_target_list(name, "upstream-healthcheck")
+        core.log.info("guohao get_healthchecker_name: ", core.json.delay_encode(upstream_target_list, true))
```

Comment:
pls remove "guohao"

Reply:
Operation error, I want to initiate PR separately.
This commit has nothing to do with Pr. I'll roll it back.